nvflare/fuel/hci/client/fl_admin_api.py

```diff
@@ -395,6 +379,23 @@ def list_jobs(self, options: str = None) -> FLAdminAPIResponse:
             APIStatus.ERROR_RUNTIME, {"message": "Runtime error: could not handle server reply."}, reply
         )
 
+    @wrap_with_return_exception_responses
+    def download_job(self, job_id: str) -> FLAdminAPIResponse:
+        if not job_id:
+            raise APISyntaxError("job_id is required but not specified.")
+        if not isinstance(job_id, str):
+            raise APISyntaxError("job_id must be str but got {}.".format(type(job_id)))
+        success, reply_data_full_response, reply = self._get_processed_cmd_reply_data("download_job " + job_id)
```

Comment:
Some of these command names you use constants like `AdminCommandNames.DELETE_RUN`

Should we use constants for all these command names?

Reply:
made an update to address some of this for some that made sense, we can continue to clean up more in the future.